taqueria-plugin-flextesa/proxy.ts

```diff
@@ -31,6 +30,8 @@ export interface Opts extends RequestArgs.ProxyRequestArgs {
 	sandboxName?: string;
 }
 
+const ECAD_FLEXTESA_IMAGE_ENV_VAR = 'TAQ_ECAD_FLEXTESA_IMAGE';
```

Comment:
I think the value for overriding our custom images is somewhat limited, but once the Flextesa plugin becomes image agnostic, this would be more valuable to end users

I'm on the fence as to whether to include `ECAD` in the variable name as this may become an override for the Oxhead Alpha images once those changes are made. Thoughts?

Reply:
Hmm not sure, but in the future users to choose to use TAQ_FLEXTESA_IMAGE so it wouldn't break backwards compatibility I think